nx/include/switch/services/set.h

```diff
@@ -175,3 +175,15 @@ Result setsysBindFatalDirtyFlagEvent(Event *out);
  * @param flags_1 Pointer to populate with second 64 flags.
  */
 Result setsysGetFatalDirtyFlags(u64 *flags_0, u64 *flags_1);
+
+/**
+ * @brief Gets the system's nickname.
+ * @param buffer Pointer to output the nickname to. (The buffer size needs to be at least 0x80 bytes)
+ */
+Result setsysGetDeviceNickname(char* buffer);
+
+/**
+ * @brief Sets the system's nickname.
+ * @param buffer Pointer to read the nickname from. (The buffer size needs to be at least 0x80 bytes)
```

Comment:
Does the buffer *really* need to be 0x80 bytes, or is this just copypasta error?